### PR TITLE
[Nano] Add Github Action Tests for How-To Guides: Inference PyTorch

### PR DESCRIPTION
**Pull request by Oscilloscope98**
## Description

Add github actions tests for Nano How-To Guides: Inference PyTorch. These guides are added through PR #5480 
### 1. Why the change?

Add tests to make sure there is no error inside the notebooks for how-to guides, and continuously check whether there would be new error when related nano source files change.

### 2. Summary of the change 

Add tests for notebooks under folder `BigDL/python/nano/tutorial/notebook/inference/pytorch`

### 3. How to test?
- [x] GitHub Action tests

Backup: we may save OcfordIIITPet dataset to our FTP server if downloading is very slow/unstable

### CODE REVIEW

**Comment by Oscilloscope98** [comment]
Not very sure why there are errors related to ipykernel when testing notebooks with openvino dependencies, as shown [here](https://github.com/intel-analytics/BigDL/runs/8241354829?check_suite_focus=true#step:6:1449) (this AttributeError did not occur for onnxruntime related tests)
```
Traceback (most recent call last):
  File "/usr/share/miniconda/envs/howto-guides-inference-pytorch-openvino/lib/python3.7/logging/__init__.py", line 2037, in shutdown
    h.close()
  File "/usr/share/miniconda/envs/howto-guides-inference-pytorch-openvino/lib/python3.7/site-packages/absl/logging/__init__.py", line 934, in close
    self.stream.close()
  File "/usr/share/miniconda/envs/howto-guides-inference-pytorch-openvino/lib/python3.7/site-packages/ipykernel/iostream.py", line 438, in close
    self.watch_fd_thread.join()
AttributeError: 'OutStream' object has no attribute 'watch_fd_thread'
```

Based on our previous implement [here](https://github.com/intel-analytics/BigDL/blob/main/.github/workflows/nano_notebooks_tests.yml#L84), I currently solve this by downgrading `ipykernel`.

File: .github/workflows/nano_howto_guides_tests.yml [hunk 1]
```diff
@@ -0,0 +1,71 @@
+name: Nano Unit Tests for How-To Guides Notebooks
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the main branch
+  push:
+    branches: [ main ]
```
[the remark this answers is not in the document]
Comment (Oscilloscope98) [reply .github/workflows/nano_howto_guides_tests.yml:7]: For other nano-related tests, it seems that no limitations are added for push on main branch either, should we also add the `paths` for other nano tests?

[the remark this answers is not in the document]
Comment (TheaperDeng) [reply .github/workflows/nano_howto_guides_tests.yml:7]: just this one for now, we need a complete overhaul on nano's action, we can leave the changes to that time.

[the remark this answers is not in the document]
Comment (Oscilloscope98) [reply .github/workflows/nano_howto_guides_tests.yml:7]: Have added, and tests has passed: https://github.com/intel-analytics/BigDL/runs/8262188106?check_suite_focus=true.

Please re-review, thank you!!

**Pull request merged**